code/src/components/MovieList.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+
+import { MovieCard } from "./MovieCard";
+
+import { MOVIES_URL } from "../reusables/urls";
+
+export const MovieList = () => {
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(MOVIES_URL)
+      .then((response) => response.json())
+      .then((json) => setMovies(json.results))
+      .catch((err) => console.error(err));
```

Comment:
Please remember to remove all console outputs from your code before submitting. 👮🏻